backend/sirius-components-diagrams-layout/src/test/java/org/eclipse/sirius/components/diagrams/layout/incremental/DiagramLayoutTest.java

```diff
@@ -90,8 +91,8 @@ private TestDiagramCreationService createDiagramCreationService(Diagram diagram)
         NodeSizeProvider nodeSizeProvider = new NodeSizeProvider(new ImageSizeProvider());
         IncrementalLayoutEngine incrementalLayoutEngine = new IncrementalLayoutEngine(nodeSizeProvider);
 
-        LayoutService layoutService = new LayoutService(new IELKDiagramConverter.NoOp(), new IncrementalLayoutDiagramConverter(), new LayoutConfiguratorRegistry(List.of()),
-                new ELKLayoutedDiagramProvider(), new IncrementalLayoutedDiagramProvider(), representationDescriptionSearchService, incrementalLayoutEngine);
+        LayoutService layoutService = new LayoutService(new ELKDiagramConverter(new TextBoundsService(), new ImageSizeProvider()), new IncrementalLayoutDiagramConverter(),
+                new LayoutConfiguratorRegistry(List.of()), new ELKLayoutedDiagramProvider(), new IncrementalLayoutedDiagramProvider(), representationDescriptionSearchService, incrementalLayoutEngine);
```

Comment:
Why is that necessary? Do you really need an ImageSizeProvider?

Reply:
Yes I need a real ELKDiagramConverter to have an non null ELKConvertedDiagram in the full layout process
But ok i will move this test to a new one out of incremental package